pkg/i2gw/providers/gce/converter.go

```diff
@@ -45,6 +46,12 @@ func newConverter(conf *i2gw.ProviderConf) converter {
 func (c *converter) convert(storage *storage) (i2gw.GatewayResources, field.ErrorList) {
 	ingressList := []networkingv1.Ingress{}
 	for _, ing := range storage.Ingresses {
+		if common.GetIngressClass(*ing) == "" {
```

Comment:
Consider adding a check for `ing != nil` to avoid panicking when the ingress is nil

Reply:
Sounds good, added. Thanks!